db/sdc/sdcSeedPostgres.js

```diff
@@ -1,7 +1,6 @@
 var fs = require('fs');
 var { Pool } = require('pg');
 var copyFrom = require('pg-copy-streams').from;
-const csv = require('csv-parser')
```

Comment:
I see you've still got the data generator here - do you just not need the CSV parser anymore?

Reply:
the csvParser was never used in the first place, I just forgot to delete it last time when I -thought- I needed it. Before, I wanted to just use one same generation script for both DB (one needed csv, the other json blobs) and using a csvParser would turn my csv into json blobs but CouchDB seemed to be more picky than just simple Json blobs and so I created a separate file for json generation